Package.swift

```diff
@@ -11,32 +11,25 @@ let package = Package(
         .macOS(.v10_12)
     ],
     products: [
-        .library(name: "ConfigCat", type: .dynamic, targets: ["ConfigCat"])
+        .library(name: "ConfigCat", targets: ["ConfigCat"]),
+        .library(name: "ConfigCatDynamic", type: .dynamic, targets: ["ConfigCat"])
```

Comment:
Thanks for the fix and the PR.
I think the explicit Dynamic product is not really needed. Could you please remove it?

```suggestion
        .library(name: "ConfigCat", targets: ["ConfigCat"]),
```

Reply:
Sounds good, removed!